.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,69 @@
+name: PR
```

Comment:
This is for PRs, would also need one to push to Docker on merge.

Reply:
I've added another action